Flashing ODROID: add info that Allen key is needed (#27079)
home-assistant/home-assistant.github.io: source/_includes/common-tasks/flashing_n2_otg.md

#### Flashing Home Assistant

1. Connect the ODROID-N2+ to your PC via the micro-USB port at the front of the ODROID-N2+. 
1. When the ODROID-N2 is recognized as a USB connected storage device, you can flash the eMMC with [Etcher](https://www.balena.io/etcher/).
   * Use the latest stable version of Home Assistant OS for the [ODROID-N2+](https://github.com/home-assistant/operating-system/releases/download/{{site.data.version_data.hassos['odroid-n2']}}/haos_odroid-n2-{{site.data.version_data.hassos['odroid-n2']}}.img.xz) (haos_odroid-n2-{{site.data.version_data.hassos['odroid-n2']}}.img.xz).

1. When the flash process is complete, disconnect the ODROID-N2+ from your PC.
   * Remove the power cable.

>    * Remove the USB and HDMI cable.

Reviewer: ```suggestion
   * Remove the USB and HDMI cables.
```

   * Make sure to toggle the boot mode switch back to MMC.

1. Put the ODROID back in its case.
1. Connect your ODROID-N2+ to your network with an Ethernet cable and plug in power.

> 1. If your router supports mDNS, you will be able to reach your installation on `http://homeassistant.local:8123`. 

Reviewer: ```suggestion
1. If your router supports mDNS, you can reach your installation at `http://homeassistant.local:8123`. 
```
